### PR TITLE
Do a bit of cleaning in the docstring

### PR DESCRIPTION
**Pull request by AlexandreAbraham**

### CODE REVIEW

File: nilearn/signal.py [hunk 1]
```diff
@@ -275,46 +275,46 @@ def butterworth(signals, sampling_rate, low_pass=None, high_pass=None,
 def high_variance_confounds(series, n_confounds=5, percentile=2.,
                             detrend=True):
     """ Return confounds time series extracted from series with highest
-        variance.
-
-        Parameters
-        ==========
```
Comment (lesteve) [inline nilearn/signal.py]: Why not replace this `=` by `-` like you did above.

Comment (lesteve) [inline nilearn/signal.py]: There are remaining `=` in other places in your diff too, that'd be great to fix them while you are at it.

File: nilearn/__init__.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
 """
 Machine Learning module for NeuroImaging in python
-==================================================
+--------------------------------------------------
```
Comment (bthirion) [inline nilearn/__init__.py]: Sorry, I'm just surprised about that one. Is there any rule somewhere about that ?
also, python -> Python

**Comment by bthirion** [comment]
Obviously good to go.

File: nilearn/__init__.py (hunk 2 above)
Comment (GaelVaroquaux) [inline nilearn/__init__.py]: > Sorry, I'm just surprised about that one. Is there any rule somewhere about
> that ?

Not that I know.

File: nilearn/input_data/nifti_spheres_masker.py [hunk 3]
```diff
@@ -1,6 +1,6 @@
 """
 Transformer for computing seeds signals.
-=======
+-------
```
Comment (GaelVaroquaux) [inline nilearn/input_data/nifti_spheres_masker.py]: That title line was too short anyhow, so it's gone from invalid to invalid; I'll fix it myself after merge.

**Comment by GaelVaroquaux** [comment]
I am merging this.

However, I am not very much in favor of such changes. These are the types of massive changes that create conflicts for people that have ongoing work on the corresponding files. And given the number of files touched, the possibilities for conflicts are quite large.

**Pull request merged**